.circleci/config.yml

```diff
@@ -19,3 +19,9 @@ jobs:
     - run: echo 0
     - run: echo 0
     - run: $HOME/ci-scripts/circleci/docker-publish $DOCKER_USER $DOCKER_PASS "$DOCKER_EMAIL" $DOCKER_ORG
+workflows:
+  build_test_publish_deploy:
+    jobs:
+    - build:
+        context:
+        - aws-ecr-public
```

Comment:
https://github.com/search?l=&q=gor-docker+user%3AClever&type=Code indicates this isn't used internally, and I couldn't find it in any of our public registries.

Technically, it was pushed to Dockerhub (https://hub.docker.com/r/clever/gor-docker/tags), but that was 5 years ago.

Kind of in the camp of graveyarding this.

Reply:
Alright. I will close this and graveyard it!